docs/framework/release-notes/2024/04-09-april-security-and-quality-rollup.md

```diff
@@ -112,3 +112,5 @@ The following table is for earlier Windows and Windows Server versions for Secur
 | .NET Framework 2.0, 3.0 | [5036633](https://support.microsoft.com/kb/5036633) |
 | .NET Framework 3.5 SP1 | [5036636](https://support.microsoft.com/kb/5036636) |
 | .NET Framework 4.6.2 | [5036631](https://support.microsoft.com/kb/5036631) |
+
+The operating system row lists a KB which will be used for update offering purposes. When the operating system KB is offered, the applicability logic will determine the specific .NET Framework update(s) will be installed. Updates for individual .NET Framework versions will be installed based on the version of .NET Framework that is already present on the device. Because of this the operating system KB is not expected to be listed as installed updates on the device. The expected update to be installed are the .NET Framework specific version updates listed in the table above.
```

Comment:
```suggestion
The operating system row lists a KB that's used for update offering purposes. When the operating system KB is offered, the applicability logic determines the specific .NET Framework update(s) that will be installed. Updates for individual .NET Framework versions are installed based on the version of .NET Framework that's already present on the device. Because of this, the operating system KB is not expected to be listed as installed updates on the device. The expected updates to be installed are the .NET Framework-specific version updates listed in the previous table.
```